web/models.py

```
@@ -2790,3 +2790,106 @@ class ScheduledPost(models.Model):
+
+class StudyGoal(models.Model):
+    """Model for storing study goals and objectives."""
+    study_plan = models.ForeignKey(StudyPlan, on_delete=models.CASCADE, related_name="goals")
+    title = models.CharField(max_length=200)
+    description = models.TextField(blank=True)
+    target_date = models.DateField()
+    completed = models.BooleanField(default=False)
+    completed_at = models.DateTimeField(null=True, blank=True)
+    priority = models.IntegerField(
+        choices=[(1, "Low"), (2, "Medium"), (3, "High")],
+        default=2
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)